staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -246,5 +246,5 @@ type Interface interface {
 // If Begin is empty, it is left unbounded.
 // If End is empty, it is right unbounded.
 type Interval struct {
-	Begin, End string
+	Begin, End, Whitelist string
```

Comment:
Does Whitelist support multiple tenants? If not, don't bother make it configurable.

Reply:
Removed